src/main/java/io/stargate/sgv2/jsonapi/service/operation/UpdateAttemptBuilder.java

```diff
@@ -24,11 +24,12 @@ public class UpdateAttemptBuilder<SchemaT extends TableSchemaObject> {
   private final SchemaT tableBasedSchema;
   private final WhereCQLClauseAnalyzer whereCQLClauseAnalyzer;
 
-  public UpdateAttemptBuilder(SchemaT tableBasedSchema) {
+  public UpdateAttemptBuilder(SchemaT tableBasedSchema, boolean updateOne) {
```

Comment:
the updateOne param appears to never be used

Reply:
yes, leave this way intentionally, cause currently we do not support updateMany for tables.
We enforce full primary key for updateOne to identify single row and update.

However, there is actually one way to do CQL 'updateMany'.
That is to use in operator. 
So we may consider adding updateMany later, more complicated surely, so leave this boolean here.